third_party/terraform/website/docs/r/compute_router_nat.html.markdown

```diff
@@ -0,0 +1,121 @@
+---
+layout: "google"
+page_title: "Google: google_compute_router_nat"
+sidebar_current: "docs-google-compute-router-nat"
+description: |-
+  Manages a Cloud NAT.
+---
+
+# google\_compute\_router\_nat
+
+Manages a Cloud NAT. For more information see
+[the official documentation](https://cloud.google.com/nat/docs/overview)
+and
+[API](https://cloud.google.com/compute/docs/reference/rest/beta/routers).
+
+## Example Usage
+
+A simple NAT configuration: enable NAT for all Subnetworks associated with
+the Network associated with the given Router.
+
+```hcl
+resource "google_compute_router_nat" "simple-nat" {
+  name                               = "nat-1"
+  router                             = "router-1"
```

Comment:
can this reference a router resource? (or do you think that people will typically configure routers outside of terraform? if that is the case, can you comment it?)

Reply:
Done.